src/isIsogram.test.js

```diff
@@ -6,4 +6,24 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`enters izogram`, () => {
```

Comment:
The description `enters izogram` is not clear. Consider renaming it to something more descriptive, like `returns true for isogram word`.